### PR TITLE
Lucah  change index

### PR DESCRIPTION
**Pull request by lucahendicott**
@ryanabooth hi. can u lookie here?

### CODE REVIEW

**Comment by lucahendicott** [comment]
Updated routes from using game index to game Id's. Right now it breaks the add question page, but will fix this with another PR after Ray sends a PR for his changes in the add question page. @RWu603 knows about this!

File: web/src/components/EditGameDialogue.jsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default function EditGameDialogue({ game, open, onClose, submit }) {
         </DialogContent>
         <DialogActions>
           <Typography variant="caption">
-            Game ID: {game.GameID}
```
Comment (ryanabooth) [inline web/src/components/EditGameDialogue.jsx:56]: I think we may want to discard this change?

**Review by ryanabooth — APPROVED** [review]
Looking good so far. You can merge this or complete the rest in this branch and then request another review.

File: web/src/components/EditGameDialogue.jsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ export default function EditGameDialogue({ game, open, onClose, submit }) {
...
+            {/* Game ID: {game.GameId} */}
```
Comment (ryanabooth) [inline web/src/components/EditGameDialogue.jsx:56]:
```suggestion
            Game ID: {game.GameID}
```

**Review by ryanabooth — APPROVED** [review]

**Pull request merged**